packages/web-api/src/methods.ts

```diff
@@ -2399,88 +2393,108 @@ export interface UsergroupsUsersUpdateArguments extends WebAPICallOptions, Token
+// https://api.slack.com/methods/users.identity
+export interface UsersIdentityArguments extends TokenOverridable { }
+// https://api.slack.com/methods/users.info
+export interface UsersInfoArguments extends TokenOverridable, LocaleAware {
   user: string;
 }
-export interface UsersListArguments extends WebAPICallOptions, TokenOverridable, CursorPaginationEnabled, LocaleAware {
-  presence?: boolean; // deprecated, defaults to false
-  team_id?: string;
-}
+// https://api.slack.com/methods/users.list
+export interface UsersListArguments extends TokenOverridable, CursorPaginationEnabled,
+  LocaleAware, OptionalTeamAssignable { }
 cursorPaginationEnabledMethods.add('users.list');
-export interface UsersLookupByEmailArguments extends WebAPICallOptions, TokenOverridable {
+// https://api.slack.com/methods/users.lookupByEmail
+export interface UsersLookupByEmailArguments extends TokenOverridable {
   email: string;
 }
-export interface UsersSetPhotoArguments extends WebAPICallOptions, TokenOverridable {
+// https://api.slack.com/methods/users.setPhoto
+export interface UsersSetPhotoArguments extends TokenOverridable {
   image: Buffer | Stream;
   crop_w?: number;
   crop_x?: number;
   crop_y?: number;
 }
-export interface UsersSetPresenceArguments extends WebAPICallOptions, TokenOverridable {
+// https://api.slack.com/methods/users.setPresence
+export interface UsersSetPresenceArguments extends TokenOverridable {
   presence: 'auto' | 'away';
 }
-export interface UsersProfileGetArguments extends WebAPICallOptions, TokenOverridable {
+// https://api.slack.com/methods/users.profile.get
+export interface UsersProfileGetArguments extends TokenOverridable {
   include_labels?: boolean;
   user?: string;
 }
-export interface UsersProfileSetArguments extends WebAPICallOptions, TokenOverridable {
+// TODO: breaking change: either profile or name/value pair must be provided
+// https://api.slack.com/methods/users.profile.set
+export interface UsersProfileSetArguments extends TokenOverridable {
   profile?: string; // url-encoded json
-  user?: string;
+  user?: string; // must be an admin user and must be on a paid plan
   name?: string; // usable if `profile` is not passed
   value?: string; // usable if `profile` is not passed
 }
 
-export interface ViewsOpenArguments extends WebAPICallOptions, TokenOverridable {
-  trigger_id: string;
+/*
+ * `views.*`
+ */
+interface BaseViewsArguments {
   view: View;
 }
-
-export interface ViewsPushArguments extends WebAPICallOptions, TokenOverridable {
+interface ViewTriggerId {
   trigger_id: string;
-  view: View;
 }
-
-export interface ViewsPublishArguments extends WebAPICallOptions, TokenOverridable {
+interface ViewInteractivityPointer {
+  interactivity_pointer: string;
+}
+// https://api.slack.com/methods/views.open
+export type ViewsOpenArguments = BaseViewsArguments & TokenOverridable & (ViewTriggerId | ViewInteractivityPointer);
```

Comment:
nice

Reply:
I think I will spend more time and just make similar types of changes for all API arguments...